admin/services/pupil.service.js

```diff
@@ -33,34 +33,31 @@ const pupilService = {
    */
   fetchSortedPupilsData: async (schoolId, sortingField, sortingDirection) => {
     // TODO: Introduce integration tests
-    try {
-      const sort = {}
-      sort[sortingField] = sortingDirection
-      return await Pupil
-        .find({
-          'school': schoolId
-        })
-        .sort(sort)
-        .lean()
-        .exec()
-    } catch (error) {
-      throw new Error(error)
-    }
+    const sort = {}
+    sort[sortingField] = sortingDirection
+    const pupils = await Pupil
+      .find({'school': schoolId})
+      .sort(sort)
+      .lean()
+      .exec()
+    return pupils
+  },
+  fetchMultiplePupils: (pupilIds) => {
+    return Pupil
+      .find({'_id': { $in: pupilIds }})
+      .exec()
   },
   /**
    * Fetches latest set of pupils answers who have completed the check.
    * @param {string} id - Pupil Id.
    */
   fetchPupilAnswers: async (id) => {
     // TODO: Introduce integration tests
-    try {
-      return await Answer.findOne({
-        pupil: mongoose.Types.ObjectId(id),
-        result: {$exists: true}
-      }).sort({createdAt: -1}).exec()
-    } catch (error) {
-      throw new Error(error)
-    }
+    const answers = await Answer.findOne({
```

Comment:
@pris54 don't you want to catch the mongoose error though if it occurs?

Reply:
@GeorgeChatzigiannis - the caller can catch any errors